### ISSUE TITLE
Accessibility of downloaded PDFs & Word docs

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1396]
The following pages allow users to download a file containing info about the activity:

- Documentation (Word doc)
- Highlight text (Word doc)
- Opinion (PDF)
- Results (PDF)

These downloaded files are likely to not be as accessible as they could be (lacking correct headings etc.)
